tensorboardX/pytorch_graph.py

```diff
@@ -52,19 +55,18 @@ def parse(graph):
         # singlequote will be escaped by tensorboard
         attrs = attrs.replace("'", ' ')
         inputs = [i.uniqueName() for i in n.inputs()]
-        # FIXME: only first output is considered (only Dropout)
-        outputnode = next(iter(n.outputs()))
-        uname = outputnode.uniqueName()
-        if outputnode.type().kind() == 'TensorType':
-            outputsize = outputnode.type().sizes()
-            nodes.append({'name': uname,
-                          'op': n.kind(),
-                          'inputs': inputs,
-                          'attr': attrs,
-                          'outputsize': outputsize})
-        else:
-            nodes.append({'name': uname, 'op': n.kind(),
-                          'inputs': inputs, 'attr': attrs})
+
+        for outputnode in iter(n.outputs()):
+            uname = outputnode.uniqueName()
+            if outputnode.type().kind() == 'TensorType':
+                outputsize = outputnode.type().sizes()
+                nodes.append({'name': uname,
+                              'op': n.kind(),
+                              'inputs': inputs,
```

Comment:
This way, multiple nodes will share the same underlying inputs array, so when we modify them below, we get the `KeyError` that the CI is failing with. The way to fix this is to initialize a seperate `inputs` array during each iteration of the loop.